internal/budgetadjustments/root.go

```diff
@@ -0,0 +1,19 @@
+package budgetadjustments
+
+import (
+	_ "embed"
+	"fmt"
+
+	"github.com/nobl9/nobl9-go/sdk"
+	"github.com/spf13/cobra"
+)
+
+func NewRootCmd(client *sdk.Client) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "budgetadjustments",
+		Short: "Manage budgetadjustment.",
```

Comment:
`Budget adjustments managment` without dot at the end

Reply:
done